update-center-source.properties

```diff
@@ -1,4 +1,4 @@
-plugins=aemrules,ansible,apigee,authaad,authgoogleoauth,authoidc,checkstyle,clover,codehawk,communitybsl,communitydelphi,communitygosu,communityrust,dependencycheck,devon4j,ecocodeandroid,ecocodejava,ecocodejavascript,ecocodephp,ecocodepython,findbugs,groovy,hadolint,ibmixaemrules,jdepend,jqassistant,l10nde,l10nfr,l10nel,l10nes,l10nit,l10nja,l10nko,l10nru,l10nzh,l10nzhtw,mutationanalysis,mybatis,openedge,pmd,shellcheck,rci,scmcvs,scmjazzrtc,scmtfvc,scmmercurial,softvis3d,sonargraphintegration,webdriver,yaml
```

Comment:
I hate to be picky, but could you put this in alpha order, please?

Reply:
you are right ! I'm picky like you :p with my reviews.
update done.